src/wollok/game.wlk

```diff
@@ -273,8 +273,8 @@ object game {
  * It is an immutable object since Wollok 1.8.0
  */
 class Position {
-  const property x = 0
-  const property y = 0
+  var property x = 0
+  var property y = 0
```

Comment:
Estaba de pasada y vengo a tirar un comentario volador. A mí me hace ruido que cambiemos a `var` porque el comentario de la clase dice "It is an immutable object since Wollok 1.8.0". Estoy de acuerdo con la idea de tener un objeto MutablePosition polimórfico con Position, pero quizás deberíamos repensar la idea de extenderlo directamente de Position y tener una jerarquía que nos permita trabajar con objetos position mutables o inmutables, o bien si la idea es volver a trabajar con positions mutables modifiquemos el wollokDoc.

Reply:
No me parece mal tener
```
       AbstractPosition
              ^
          ____|___________
         |                |
   Position         MutablePosition        
```